src/integrationTest/java/uk/gov/hmcts/reform/translate/controllers/DictionaryControllerIT.java

```diff
@@ -113,4 +121,116 @@ void shouldReturn200WhenLegalRequestIsSubmitted() throws Exception {
         }
     }
+
+    @Nested
+    @DisplayName("Put Dictionary")
+    class PutDictionary {
+        private final String serviceJwtDefinition = generateDummyS2SToken("ccd_definition");
+
+        private final String serviceJwtXuiWeb = generateDummyS2SToken("xui_webapp");
+
+        // manage-translations
+        @Test
+        @Sql(scripts = {DELETE_TRANSLATION_TABLES_SCRIPT})
+        void shouldReturn201ForPutDictionaryForIdamMUserWithManageTranslationCreateANewRecord() throws Exception {
+
+            stubUserInfo("manage-translations");
+            mockMvc.perform(put(DICTIONARY_URL)
+                                .header("ServiceAuthorization", serviceJwtXuiWeb)
+                                .contentType(MediaType.APPLICATION_JSON_VALUE)
+                                .content(objectMapper.writeValueAsString(getDictionaryRequest(1, 2))))
+                .andExpect(status().is(201))
+                .andReturn();
+        }
+
+        @Test
+        @Sql(scripts = {DELETE_TRANSLATION_TABLES_SCRIPT,PUT_CREATE_ENGLISH_PHRASES_WITH_TRANSLATIONS_SCRIPT})
+        void shouldReturn201ForPutDictionaryForIdamUserWithManageTranslationUpdateARecord() throws Exception {
+            stubUserInfo("manage-translations");
+            mockMvc.perform(put(DICTIONARY_URL)
+                                .header("ServiceAuthorization", serviceJwtXuiWeb)
+                                .contentType(MediaType.APPLICATION_JSON_VALUE)
+                                .content(objectMapper.writeValueAsString(getDictionaryRequest(1, 2))))
+                .andExpect(status().is(201))
+                .andReturn();
+        }
+
+
+        // load-translations user
+        @Test
+        void shouldReturn201ForPutDictionaryForIdamUserWithLoadTranslation() throws Exception {
+
+            stubUserInfo("load-translations");
+            mockMvc.perform(put(DICTIONARY_URL)
+                                .header("ServiceAuthorization", serviceJwtXuiWeb)
+                                .contentType(MediaType.APPLICATION_JSON_VALUE)
+                                .content(objectMapper.writeValueAsString(getDictionaryRequestWithoutABody(1, 2))))
+                .andExpect(status().is(201))
+                .andReturn();
+
+        }
+
+        @Test
+        @Sql(scripts = {DELETE_TRANSLATION_TABLES_SCRIPT,PUT_CREATE_ENGLISH_PHRASES_WITH_TRANSLATIONS_SCRIPT})
+        void shouldReturn201ForPutDictionaryForIdamUserWithLoadTranslationUpdateARecord() throws Exception {
+
+            stubUserInfo("load-translations");
+            mockMvc.perform(put(DICTIONARY_URL)
+                                .header("ServiceAuthorization", serviceJwtXuiWeb)
+                                .contentType(MediaType.APPLICATION_JSON_VALUE)
+                                .content(objectMapper.writeValueAsString(getDictionaryRequestWithoutABody(1, 2))))
+                .andExpect(status().is(201))
+                .andReturn();
+        }
```

Comment:
Technically this is a no action, rather than an update.  Should it be renamed?

Reply:
You could rename both load-translation tests:
* shouldReturn201ForPutDictionaryForIdamUserWithLoadTranslation -> shouldReturn201ForPutDictionaryForIdamUserWithLoadTranslation**NewPhrases**
* shouldReturn201ForPutDictionaryForIdamUserWithLoadTranslationUpdateARecord -> shouldReturn201ForPutDictionaryForIdamUserWithLoadTranslation**ExistingPhrases**

or if doing verify step: the later could be: shouldReturn201ForPutDictionaryForIdamUserWithLoadTranslation**ExistingPhrasesPreservesExistingTranslations**

Reply:
Could We add the recommended name next to each function  next time ?

Reply:
Resolved in 5e7cdedb0ea9f964691a8eed7564fc462cd71ab8